.ci/scripts/diff_dswx_files.py

```diff
@@ -0,0 +1,139 @@
+#   diff_dswx_files.py
+#
+#   For: OPERA project
+#
+#   11-03-2023 - Ray Bambery
+#       initial release
```

Comment:
Change history such as this is captured within the commit history in git itself, so you do not need to document it within the code itself.